385/Assets/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Assets.Scripts;
+
+public class PlayerCamera : MonoBehaviour {
+    // player object camera is attached to
+    public GameObject player;
+
+    // offset of camera from player
+    public Vector3 offset;
+
+    // Static position of camera
+    private Vector3 initPos;
+    private Vector3 playerInit;
+
+    // camera that follows player
+    public Camera playerCam;
+
+    // checks Camera1 to change camera settings
+    public string CameraAxis = "Camera1";
+
+    // used to check is camera button is pushed
+    private AxisButton CameraButton;
+
+    // tracks whether camera is focused on the player
+    public bool trackPlayer = false;
+
+    // adjusts movement speed of camera
+    public float camMoveSpeed = 1;
+
+    // distance and X and Y vector offset of camera
+    public float offsetDistance = 2, camXoffset, camYoffset;
+                                  
+    // tracks whether camera is suppose to be moving
+    private bool camMoving = false;
+
+    public RopeSystem RopeSystem;
+
+    // gets offset for camera and starts camera in static view
+    void Start () {
+        camYoffset = 1;
+        Vector3 temp = new Vector3(-camXoffset, camYoffset, offsetDistance);  // stores adjustments
+        offset = transform.position - player.transform.position + temp;
+
+        initPos = transform.position;  // stores static camera position
+        playerInit = player.transform.position;
+
+        // camera axis button created
+        CameraButton = new AxisButton(CameraAxis, 0.5f);
+    }
+	
+    // updates camera button and camera position
+	void LateUpdate () {
+        Vector3 camtemp = new Vector3(-camXoffset, camYoffset, offsetDistance);  // stores adjustments
```

Comment:
why are these three values? Why can't I just set the offset of the camera using a Vector3